package.json

```diff
@@ -32,8 +32,8 @@
   },
   "scripts": {
     "prepublish": "babel src --out-dir dist",
-    "lint": "eslint .",
-    "tape": "babel-tape-runner 'src/**/__tests__/*.js'",
+    "lint": "eslint src",
```

Comment:
I think that saves us having the duplicate anything and not worry about symlinks on windows?